runfile/sleep/README.md

```diff
@@ -0,0 +1,2 @@
+# Sleep benchmark
+A benchmark born to sleep
```

Comment:
Might want to add something here about why you would want to use this: ie. to collect data for an already running workload.